packages/react-slider/src/utils/clamp.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Validates that the `value` is a number and falls between the min and max.
```

Comment:
This comment seems wrong based on the implementation.

Also I wonder if this should live in `react-utilities`?
```suggestion
 * Clamps `value` to a number between the min and max.
```